rpmautospec/pkg_history.py

```diff
@@ -123,15 +123,33 @@ def _get_rpmverflags(cls, path: str, name: Optional[str] = None) -> Optional[str
             "--qf",
             query,
             "--specfile",
-            f"{name}.spec",
-        ]
-
-        call = subprocess.run(
-            rpm_cmd,
-            cwd=path,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
         )
+
+        with specfile.open(mode="rb") as unabridged, NamedTemporaryFile(
+            mode="wb", prefix=f"rpmautospec-abridged-{name}-", suffix=".spec", delete=False
```

Comment:
Just curious. When is this file deleted?

Reply:
Never, good catch! It’s a piece I just put in for debugging and forgot to take it out again. 😬